lib/ansible/module_utils/crypto.py

```diff
@@ -63,7 +63,7 @@ def load_privatekey(path, passphrase=None):
         if passphrase:
             privatekey = crypto.load_privatekey(crypto.FILETYPE_PEM,
                                                 open(path, 'rb').read(),
-                                                passphrase)
+                                                passphrase.encode('UTF-8'))
```

Comment:
@pilou- By reading this answer I'd say `encode()` seems the way to go https://stackoverflow.com/questions/7585435/best-way-to-convert-string-to-bytes-in-python-3